GitUI/CommandsDialogs/FormBrowse.cs

```diff
@@ -660,11 +660,11 @@ private void RefreshSelection()
             FillFileTree(selectedRevision);
             FillDiff(selectedRevisions);
 
-            string oldBody = selectedRevision?.Body;
+            string oldBody = (selectedRevision != null ? UIExtensions.FormatBodyAndNotes(selectedRevision.Body, selectedRevision.Notes) : null);
```

Comment:
Just compare Body and Notes separately.
I question that the check is needed at all. Maybe this is when the full body is fetched and Notes can be updated, but may be overkill.